clippy_lints/src/methods/mod.rs

```diff
@@ -312,6 +312,26 @@ declare_clippy_lint! {
     "using combination of `filter_map` and `next` which can usually be written as a single method call"
 }
 
+declare_clippy_lint! {
+    /// **What it does:** Checks for usage of `flat_map(|x| x)`.
+    ///
+    /// **Why is this bad?** Readability, this can be written more concisely by using `flatten`.
```

Comment:
I think you can remove the word "readability"

```suggestion
    /// **Why is this bad?** This can be written more concisely by using `flatten`.
```